### PR TITLE
Add continue-on-error boolean to Pipeline Nodes

### PR DESCRIPTION
**Pull request by tokkoro**
Add optional continue-on-error boolean to each node for https://github.com/valohai/roi/issues/3577

### CODE REVIEW

File: valohai_yaml/objs/pipelines/deployment_node.py [hunk 1]
```diff
@@ -20,8 +20,9 @@ def __init__(
         actions: Optional[List[NodeAction]] = None,
         endpoints: Optional[List[str]] = None,
         aliases: Optional[List[str]] = None,
+        continue_on_error: Optional[bool] = False,
```
Comment (akx) [inline valohai_yaml/objs/pipelines/deployment_node.py:23]: The value can't be true/false/none, can it?

```suggestion
        continue_on_error: bool = False,
```

File: valohai_yaml/objs/pipelines/execution_node.py [hunk 2]
```diff
@@ -17,9 +17,10 @@ def __init__(
         name: str,
         step: str,
         actions: Optional[List[NodeAction]] = None,
-        override: Optional[NodeOverrideDict] = None
+        override: Optional[NodeOverrideDict] = None,
+        continue_on_error: Optional[bool] = False,
```
Comment (akx) [inline valohai_yaml/objs/pipelines/execution_node.py:21]:
```suggestion
        continue_on_error: bool = False,
```

File: valohai_yaml/objs/pipelines/node.py [hunk 3]
```diff
@@ -20,14 +20,19 @@ class Node(Item):
...
+    # `continue_on_error` if pipeline execution should continue even when node was erroneous
+    continue_on_error: bool
+
     def __init__(
         self,
         *,
         name: str,
-        actions: Optional[List[NodeAction]] = None
+        actions: Optional[List[NodeAction]] = None,
+        continue_on_error: Optional[bool] = False
```
Comment (akx) [inline valohai_yaml/objs/pipelines/node.py:31]:
```suggestion
        continue_on_error: bool = False
```

**Review by akx — CHANGES_REQUESTED** [review]
Type question.

**Review by akx — APPROVED** [review]

**Comment by tokkoro** [comment]
Closing this as #66 will override it.

**Pull request closed without merging**